storage/innobase/handler/ha_innodb.cc

```diff
@@ -15013,7 +15013,8 @@ ha_innobase::info_low(
 				index selectivity is 2 times better than
 				our estimate: */
 
-				rec_per_key_int = rec_per_key_int / 2;
+                                if (thd_double_innodb_cardinality(m_user_thd))
+					rec_per_key_int = rec_per_key_int / 2;
```

Comment:
The first line is being indented with spaces, while the second line is indented by TABs. I would prefer arithmetics to conditional evaluation, to be more friendly towards modern superscalar CPU cores:
```c++
				rec_per_key_int /= 1
					+ thd_double_innodb_cardinality(m_user_thd);
```
That corresponds to the archaic original InnoDB formatting that this function is in.